app/pages/error/__snapshots__/template.test.js.snap

```diff
@@ -19,6 +19,7 @@ exports[`error page should render the content 1`] = `
     <h1 class=\\"nhsuk-heading-l nhsuk-u-margin-top-5\\" data-test-id=\\"error-title\\">Error</h1>
     <p data-test-id=\\"error-description\\">Document not found</p>
     
+    
```

Comment:
Extra line here?

Reply:
Yeah, not sure why that was required – don't think it implies any real issues though due to white space treatment in HTML. Also isolated to the error page. We can always revisit if we do encounter any obvious visual issues though.